webui/react/src/pages/F_ExpList/glide-table/custom-renderers/cells/linkCell.tsx

```diff
@@ -87,6 +93,29 @@ const renderer: CustomRenderer<LinkCell> = {
     }
     ctx.fillStyle = theme.linkColor;
     ctx.fillText(link.title, drawX, drawY);
+    if (link.unmanaged) {
+      const x = drawX + commaMetrics.width + 8;
+      const y = drawY - TAG_HEIGHT / 2;
+      ctx.fillStyle = '#132231';
+      ctx.lineWidth = 2;
+      ctx.strokeStyle = theme.textBubble;
+      ctx.beginPath();
+      const tagFont = `600 11px ${theme.fontFamily}`;
```

Comment:
**nit suggestion:** in javascript, there are a lot of hoisting going on (where variable and function declarations are moved to the top) so in code it's common practice to have all the declarations like `const x = y` etc all grouped together at the top of the block whenever possible. There are exceptions such as when the declaration has a dependency on another runtime code, needing it to follow after that execution.

Reply:
Cool, that's great suggestion, I grouped them together under `if (link.unmanaged)` to make it obvious that some values are only related to unmanaged experiments. 
Regarding not able to view the unmanaged tag for experiment list table, have you rebuilt the backend from this branch?